src/librustc_yk_sections/mir_cfg.rs

```diff
@@ -454,9 +463,10 @@ impl<'tcx> ToPack<ykpack::Rvalue> for (&ConvCx<'_, 'tcx, '_>, &Rvalue<'tcx>) {
     }
 }
 
-/// At the time of writing, you can't pop from a `HashSet`.
-fn hashset_pop<T>(h: &mut FxHashSet<T>) -> T where T: Eq + Hash + Clone {
-    //let e: () = h.iter().cloned().next().unwrap();
-    let e = h.iter().next().unwrap().clone();
-    h.take(&e).unwrap()
+/// At the time of writing, you can't pop from a `BitSet`.
+fn bitset_pop<T>(s: &mut BitSet<T>) -> T where T: Eq + Idx + Clone {
+    let e = s.iter().next().unwrap().clone();
```

Comment:
Does `iter` give you the next set bit, or just the next bit?

Reply:
The docstring says:
> Iterates over the indices of set bits in a sorted order.

Which I think is correct. Do you agree?

Reply:
Agreed.